### DISCUSSION TITLE
yaml parse exception

### DISCUSSION THREAD

**Discussion by remipoupard** [jgm/pandoc#9503] [Q&A]
Hi, I'm a 18 info student and we have to answer our homework using Markdown.
While trying to convert it in pdf with Pandoc, I got this error

$pandoc -f markdown -t pdf Configuration.md
YAML parse exception at line 3, column 1,
while scanning an alias:
did not find expected alphabetic or numeric character

Here's my markdown, can you help me please I don't understand this error.

[Configuration.md](https://github.com/jgm/pandoc/files/14392779/Configuration.md)

**Comment by kysko** [comment]
From the user guide on [Horizontal rules](https://pandoc.org/MANUAL.html#horizontal-rules):

> We strongly recommend that horizontal rules be separated from surrounding text by blank lines. If a horizontal rule is not followed by a blank line, pandoc may try to interpret the lines that follow as a YAML metadata block or a table.

This is what happens on lines 69-70 of your file. So try adding a blank line after the `---`.
Or reconsider your use of the horizontal rules.

(I'd have other recommendations for your use of `markdown` in this file, but it's beyond your question.)

**Reply by remipoupard** [reply]
Thank you very much !!
Could you please give me your other recommendations ?
